linters/buildifier/buildifier.test.ts

```diff
@@ -1,5 +1,32 @@
-import { linterCheckTest, linterFmtTest } from "tests";
+import path from "path";
+import { customLinterFmtTest, linterCheckTest } from "tests";
+import { TrunkDriver } from "tests/driver";
+import { TEST_DATA } from "tests/utils";
+
+const preFmt = (driver: TrunkDriver) => {
+  const buildifierTablesContents = `
+{
+  "IsLabelArg": {},
+  "LabelDenylist": {},
+  "IsSortableListArg": {
+    "fizz": true
+  },
+  "SortableAllowlist": {},
+  "NamePriority": {}
+}
+  `;
+
+  driver.moveFile(".buildifier.json", ".trunk/configs/.buildifier.json");
+  driver.deleteFile(".buildifier-tables.json");
+  driver.writeFile(".trunk/configs/.buildifier-tables.json", buildifierTablesContents);
+};
 
 // TODO(Tyler): We will eventually need to add a couple more test cases involving failure modes.
 linterCheckTest({ linterName: "buildifier" });
-linterFmtTest({ linterName: "buildifier" });
+customLinterFmtTest({
+  linterName: "buildifier",
+  args: "-a",
+  testName: "add_tables",
+  pathsToSnapshot: [path.join(TEST_DATA, "add_tables.in.BUILD")],
```

Comment:
Can you append `path.join(TEST_DATA, "basic.in.bzl")` as well and also audit `buildifier_v4.0.1_basic.fmt.shot`? Looks like we lost some coverage here.

Because of the way we use `jest-specific-snapshot`, unfortunately it doesn't auto-detect the fact that we lost coverage.